ruby_on_rails/rails_sprinkles/js_bundling.md

```diff
@@ -139,14 +139,13 @@ With that you will be able to add your controllers to the bundling process and u
 
 ### Why Use JS Bundling
 
-You may be wondering if we have import maps why we need JS bundling at all. One of the _most_ important things to remember is that this decision 
-depends on the circumstances of your project and that while import maps does have the benefits of not needing to bundle or transpile you lose a 
-few benefits you would have by using JS bundling. Some frameworks like React which require compiling for JSX does not work with import maps which
-leads to another important point. The JavaScript system is a large ecosystem upon which bundling was introduced to reduce performance costs associated with HTTP 1. Being the case that HTTP 1 supported a limited amount of requests on a single connection, introducing many files carried a performance cost. While HTTP 2 reduced those performance costs much of the JS ecosystem _still_ requires explicit transpiling or bundling due to packages being built on the need for bundling for performance or a framework simply requiring it anyways. For your project if you want to use some of these Frameworks you may want to reach for JS bundling over import maps.
+You may be wondering why we even need JS bundling when we have import maps. One of the _most_ important things to remember is that this decision 
```

Comment:
Nit: The markdown has weird line breaks here so you can't directly commit this suggestion, but this overall block of text could use breaking up too: 

```suggestion
You may be wondering why we even need JS bundling when we have import maps. One of the _most_ important things to remember is that this decision  depends on the circumstances of your project and that while import maps eliminates the need for bundling and transpiling, you lose out on some of the benefits of using JS bundling. 

Some frameworks like React, which require compiling for JSX, do not work with import maps which
leads to another important point. The JavaScript ecosystem is vast, and bundling was introduced to mitigate performance costs associated with HTTP 1. Because HTTP 1 limited the number of requests per connection, sending many files carried a performance cost. 

Although HTTP 2 has reduced these costs, many JS packages still require explicit transpiling or bundling for performance or because of framework requirements. If you plan to use such frameworks for your project, it may be best to choose JS bundling over import maps.
```

Reply:
Okay, thanks Kevin and sorry I took this long to respond. I'm still not used to this PR workflow. I guess I'm supposed to "Resolve Conversation", so I'll do that now.